pkgs/applications/networking/irc/weechat/default.nix

```diff
@@ -18,64 +19,92 @@ assert tclSupport -> tcl != null;
 
 let
   inherit (pythonPackages) python pycrypto pync;
-in
+  weechat =
+    stdenv.mkDerivation rec {
+      version = "1.7.1";
+      name = "weechat-${version}";
 
-stdenv.mkDerivation rec {
-  version = "1.7.1";
-  name = "weechat-${version}";
+      # TODO: use next stable version
+      src = fetchFromGitHub {
+        owner = "lheckemann";
+        repo = "weechat";
+        rev = "546e260fd58e41db20e7bc4ec5585e93fb6994c1";
```

Comment:
No, it's master as of yesterday + my patch. I've changed it back to use v1.7.1 plus my patch
https://github.com/NixOS/nixpkgs/pull/25274/commits/368edaf44f4224e8699b9c66035716ae892d0fe2